cli.js

```diff
@@ -69,4 +69,7 @@ if (cli.flags.replacement === undefined) {
 		ignoreCase: cli.flags.ignoreCase,
 		glob: cli.flags.glob
 	});
-})();
+})().catch(error => {
+	console.error(error);
+	process.exit(1);
+});
```

Comment:
This is moot. `meow` handles this for us.

Reply:
I don't remember the exact issue, but my commit note says that this is needed to fix a failing test ('no globs').

Reply:
It's been a few weeks... I don't see the error now. 
I'll drop the commit.